CombinatorialGames/Game/Short.lean

```diff
@@ -72,6 +75,15 @@ protected theorem isOption [Short x] (h : IsOption y x) : Short y := by
 
 alias _root_.IGame.IsOption.short := Short.isOption
 
+protected theorem subposition {x : IGame} [Short x] (h : Subposition y x) : Short y := by
```

Comment:
```suggestion
protected theorem subposition {x y : IGame} [Short x] (h : Subposition y x) : Short y := by
```

Reply:
`y` already exists as an implicit variable. So does `x`, but we need to specify it explicitly so that it's generalized in our induction.